packages/ui-components/src/assets/svg-symbols.svg

```diff
@@ -1138,6 +1138,10 @@
 		<path d="M8 1.333A4.663 4.663 0 0 0 3.333 6C3.333 9.5 8 14.667 8 14.667S12.667 9.5 12.667 6A4.663 4.663 0 0 0 8 1.333zm0 6.334a1.667 1.667 0 1 1 .001-3.335A1.667 1.667 0 0 1 8 7.667z" />
 	</symbol>
 
+	<symbol id="kv-double-arrow-up" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
+		<path d="M6.16667 15.3333L5 14.1667L10 9.16667L15 14.1667L13.8333 15.3333L10 11.5208L6.16667 15.3333ZM6.16667 10.3333L5 9.16667L10 4.16667L15 9.16667L13.8333 10.3333L10 6.52084L6.16667 10.3333Z" />
```

Comment:
This is parsed by svgo?

Reply:
It is now, thanks for the heads up :pray: